Vale cleanup (#4151)
postmanlabs/postman-docs: CONTRIBUTING.md

When contributing to this repository, please first discuss the change you wish to make at [Issues](https://github.com/postmanlabs/postman-docs/issues).

> Please note there is a [code of conduct](https://www.postman.com/code-of-conduct); please follow it in all your interactions with the project.

Reviewer: Maybe an entire reword to avoid using "please" 2x plus a semicolon?

 ```suggestion
We have a [code of conduct](https://www.postman.com/code-of-conduct) that you're expected to follow in all your interactions with the project.
```
